### PR TITLE
Some metrics improvement and cleaning

### PR DESCRIPTION
**Pull request by Morriar**

### CODE REVIEW

**Comment by privat** [comment]
+1 even if you did not use the sort facilities on maps (`values_sorted_by_key` or `keys_sorted_by_values`) cf.

http://nitlanguage.org/doc/stdlib/module_standard__collection__sorter.html#standard__collection__abstract_collection

**Comment by jpages** [comment]
+1

**Comment by Morriar** [comment]
@privat, I set it again in need_review, I will reroll with the Map sorter.

**Comment by Morriar** [comment]
Reroll with `keys_sorted_by_values`

File: src/metrics/metrics_base.nit [hunk 1]
```diff
@@ -211,6 +211,19 @@ interface Metric
...
+		return values.keys_sorted_by_values(new DescMetricComparator)
+	end
+end
+
+private class DescMetricComparator
+	super Comparator
+
+	redef type COMPARED: Comparable
+
+	redef fun compare(a, b) do return b <=> a
```
Comment (privat) [inline src/metrics/metrics_base.nit]: this is the `reverse_default_comparator` ? Maybe just introduce it in lib/standard/collection/sorter.nit

**Comment by Morriar** [comment]
Reroll with the `DefaultReverseCompartor` in lib. Also fixed some missing comments and cleaned some code.

**Comment by privat** [comment]
+1

**Pull request merged**